src/error.rs

```diff
@@ -230,8 +230,8 @@ impl From<tokio::time::Elapsed> for Error {
 }
 
 impl From<url::ParseError> for Error {
-    fn from(_: url::ParseError) -> Error {
-        let kind = ErrorKind::DatabaseUrlIsInvalid("Error parsing database connection string.".to_string());
+    fn from(e: url::ParseError) -> Error {
+        let kind = ErrorKind::DatabaseUrlIsInvalid(format!("{}", e));
```

Comment:
nitpick: clippy would advise replacing the `format!("{}", e)` with `e.to_string()`.